src/Services/Air/validators/reservation-locator.js

```diff
@@ -0,0 +1,8 @@
+import { AirValidationError } from '../AirErrors';
+
+export default (params) => {
+  if (params.uapi_reservation_locator === undefined
```

Comment:
I'm not sure it's exactly base64, maybe we just should check if it's string and has length more than one?